kolibri/core/assets/src/views/language-switcher/mixin.js

```diff
@@ -1,11 +1,12 @@
 import { redirectBrowser } from 'kolibri.utils.browser';
 import { availableLanguages, currentLanguage } from 'kolibri.utils.i18n';
 import { httpClient } from 'kolibri.client';
+import urls from 'kolibri.urls';
 
 export default {
   methods: {
     switchLanguage(code) {
-      const path = this.Kolibri.urls['kolibri:core:set_language']();
+      const path = urls['kolibri:core:set_language']();
```

Comment:
could this be accessed through plugin data instead of aliasing through api spec?

Reply:
It could - but `pluginData` is scoped to the specific frontend bundle you are in, so it would only be accessible in this way inside the default frontend bundle. It seems easier and more consistent to continue to use the `kolibri.urls` import throughout Kolibri.